roles/infrastructure/defaults/main.yml

```diff
@@ -91,7 +92,10 @@ infra__aws_vpc_az_count:            "{{ infra.aws.vpc.az_count | default(3) }}"
 infra__aws_igw_name:                "{{ infra.aws.vpc.internet_gateway.name | default([infra__namespace, infra__aws_igw_suffix] | join('-')) }}"
 infra__aws_igw_suffix:              "{{ infra.aws.vpc.internet_gateway.suffix | default(common__igw_suffix) }}"
 infra__aws_route_table_name:        "{{ infra.aws.vpc.labels.route_table | default([infra__namespace, infra__aws_route_table_suffix] | join('-')) }}"
```

Comment:
For consistency, this variable should reflect that it is the _public_ route table, and a new variable added for a label, defaulting to "pub".

Reply:
This is the default route table. We are leveraging it as public route table. Let me fix this.